stock_indicators/indicators/ichimoku.py

```
@@ -15,14 +15,14 @@ def get_ichimoku(quotes: Iterable[Quote], tenkan_periods: int = 9,
+                 kijun_periods: int, senkou_b_periods: int, *,
                  offset_periods: int) -> "IchimokuResults[IchimokuResult]": ...
 @overload
 def get_ichimoku(quotes: Iterable[Quote], tenkan_periods: int,
-                 kijun_periods: int, senkou_b_periods: int,
+                 kijun_periods: int, senkou_b_periods: int, *,
                  senkou_offset: int, chikou_offset: int) -> "IchimokuResults[IchimokuResult]": ...
-def get_ichimoku(quotes: Iterable[Quote], tenkan_periods: int = None,
-                 kijun_periods: int = None, senkou_b_periods: int = None,
+def get_ichimoku(quotes: Iterable[Quote], tenkan_periods: int = 9,  # pylint: disable=too-many-positional-arguments
+                 kijun_periods: int = 26, senkou_b_periods: int = 52,
                  senkou_offset: int = None, chikou_offset: int = None):
```

Ruff (0.14.1)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)